ci/pipeline.yml

```diff
@@ -0,0 +1,134 @@
+---
+jobs:
+- name: logsearch-s3archiving-check-b.cld.gov.au
+  serial: true
+  plan:
+  - do:
+    - get: 5m
+      trigger: true
+    - task: check-bucket
+      params:
+        BUCKET_NAME: {{b-cld-bucket-name}}
+        AWS_ACCESS_KEY_ID: {{aws-access-key-id}}
+        AWS_SECRET_ACCESS_KEY: {{aws-secret-access-key}}
+        AWS_DEFAULT_REGION: {{aws-default-region}}
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: govau/cga-cf-bosh-cli
+        run:
+          path: sh
+          args:
+          - -exc
+          - aws s3api list-objects --bucket $BUCKET_NAME --prefix $(date -u --date="-20 minutes" +%Y/%m/%d/%H/%M) | grep Size
+    # on_failure:
+    #   put: slack
+    #   params:
+    #     text: |
+    #       :x: $BUILD_JOB_NAME FAILED
+    #       <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
```

Comment:
I think you accidentally forgot to uncomment this? (and more below)

Reply:
I've removed the slack notification where not required.